dataeng/resources/setup-course-exporter.sh

```diff
@@ -0,0 +1,32 @@
+# Create destination directory
+WORKING_DIRECTORY=/mnt/ephemeral-01/analytics-course-exporter
+mkdir -p ${WORKING_DIRECTORY}/course-data
+
+# Install requirements into this (exporter) virtual environment
+pushd analytics-exporter/
+pip install -r github_requirements.txt
+pip install --allow-external mysql-connector-python -e .
```

Comment:
As I think you and Andrew already noted, the --allow-external option is no longer used.  We pulled it from pipeline makefile in Jan. 2017:  https://github.com/edx/edx-analytics-pipeline/commit/fcebbb7859be66c54d4fa3e30e50f67ed20278e1#diff-b67911656ef5d18c4ae36cb6741b7965.

Reply:
Done.